Ginger/Ginger/UserControlsLib/UCComboBox.xaml.cs

```diff
@@ -80,14 +80,14 @@ public void Init(object bindedObject, string bindedObjField, Type optionalEnumTy
             BindVEAndSelectionChangedEvent(false, UCselectionChange);
         }
 
-        public void Init(object bindedObject, string AttrName, dynamic comboBoxEnumItemsList)
+        public void Init(object bindedObject, string AttrName, dynamic comboBoxEnumItemsList, DependencyProperty dependencyProperty)
```

Comment:
defualt value for dependencyProperty must set to ComboBox.SelectedValueProperty to keep existing behavior as is in other references of this method.

Reply:
can't have default value to this property